fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -74,7 +74,9 @@ const isQuestionInvalid = (question: NewQuestion): boolean =>
  * @param questions The list of questions
  */
 const haveQuestionsSameTitle = (questions: NewQuestion[]): boolean => {
-  const questionTitles = questions.map((q: NewQuestion) => q.question);
+  const questionTitles = questions
+    .map((q: NewQuestion) => q.question.trim())
+    .filter((string) => string !== '');
```

Comment:
Again, this should be done by the user as this is not something one would expect to happen automatically.